moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -2039,28 +2039,40 @@ double RobotState::testRelativeJointSpaceJump(const JointModelGroup* group, std:
 double RobotState::testAbsoluteJointSpaceJump(const JointModelGroup* group, std::vector<RobotStatePtr>& traj,
                                               double revolute_threshold, double prismatic_threshold)
 {
-  bool test_revolute = revolute_threshold > 0;
-  bool test_prismatic = revolute_threshold > 0;
+  struct
+  {
+    double limit;
```

Comment:
copying this again is not memory efficient

Reply:
I'm not sure about your comment. 
Are you really complaining about copying a single double?
The idea was too provide both, the limit and the bool, in an array to directly refer to them by a single index.